dds/DCPS/RTPS/Sedp.cpp

```diff
@@ -1344,12 +1316,12 @@ Sedp::send_builtin_crypto_tokens(
   const DCPS::RepoId dst = make_id(dstParticipant, dstEntity);
   if (DCPS::GuidConverter(src).isReader()) {
     create_and_send_datareader_crypto_tokens(
```

Comment:
I don't understand the purpose of listing the arguments this way. Could you move the first argument up to this line? Same with the datawriter case?